db/seeds.rb

```diff
@@ -44,12 +44,14 @@ def build_menus_with_dishes(menu_names)
 
 def add_dish(number, menu, restaurant)
   price = rand(20..60)
-  prefix = restaurant.cuisine
+  prefix = restaurant.category
   Dish.create(name: "#{prefix} Dish #{number}",
               price: price,
-              description: "A great #{restaurant.cuisine} dish!",
-              menu: menu,
-              category: Dish::CATEGORIES.sample)
+              description: "A great #{restaurant.category} dish!",
+              # category: Dish::CATEGORIES.sample
```

Comment:
I just figure we'll have to re-seed when it changes... otherwise I'll have to write the categories in my script too. Or maybe I should do that and grab the list from them?